src/components/SplitAndCounter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux';
+import { mapTreatmentToProps, mapIsFeatureOnToProps } from '@splitsoftware/splitio-redux'
+
+/**
+ * This example showcasts `mapTreatmentToProps` and `mapIsFeatureOnToProps` functions.
+ */
+
+function SplitAndCounter({ counter, isReady, feature, isFeatureOn }) {
+  return (
+    <div>
+      <p>{isReady ? 'SDK is ready' : 'SDK is not ready'}</p>
+      <p>{`test_split is ${feature}`}</p>
+      <p>{`is test_split 'on' ? ${isFeatureOn}`}</p>
+      <p>{counter}</p>
+    </div>
+  )
+}
+
+const mapTestSplitToProps = mapTreatmentToProps('test_split');
+const mapIsTestSplitToProps = mapIsFeatureOnToProps('test_split');
+
+const mapStateToProps = (state) => {
+  return {
+    counter: state.counter,
+    isReady: state.splitio.isReady,
+    ...mapTestSplitToProps(state),
+    ...mapIsTestSplitToProps(state),
```

Comment:
Add a comment next to each line explaining a bit that the mapIsFeatureOnToProps is returning a **boolean** if the treatment for test_split matches `on` specifically. The other one actually maps the treatment string. 

See if you have a way to show on the code snippet how to consume that string too (or at least the assigned name, mentioning in a code comment should be enough)